documentation/src/main/asciidoc/topics/rest_examples/rest_full_cache_detail.json

```diff
@@ -0,0 +1,48 @@
+{
+  "stats": {
+    "misses": -1,
+    "timeSinceStart": -1,
+    "timeSinceReset": -1,
+    "hits": -1,
+    "currentNumberOfEntries": -1,
+    "currentNumberOfEntriesInMemory": -1,
+    "totalNumberOfEntries": -1,
+    "stores": -1,
+    "offHeapMemoryUsed": -1,
+    "dataMemoryUsed": -1,
+    "retrievals": -1,
+    "removeHits": -1,
+    "removeMisses": -1,
+    "evictions": -1,
+    "averageReadTime": -1,
+    "averageReadTimeNanos": -1,
+    "averageWriteTime": -1,
+    "averageWriteTimeNanos": -1,
+    "averageRemoveTime": -1,
+    "averageRemoveTimeNanos": -1,
+    "requiredMinimumNumberOfNodes": -1
+  },
+  "size": 0,
+  "configuration": {
+    "template": false,
+    "elementDefinition": {
+      "topLevel": true
```

Comment:
What is this topLevel? Is this a real response? Something seems wrong...

Reply:
Ths is what I get from configuration. I was about to ask you